portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -0,0 +1,51 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.Comment;
+import com.google.sps.data.Conversation;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comments")
+public class CommentServlet extends HttpServlet {
+
+  private Conversation converse = new Conversation();
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json");
+    String json = new Gson().toJson(converse);
+    response.getWriter().println(json);
+  }
+
+  @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // If the user sends another POST request after the game is over, then start a new game.
+    String username = getUsername(request);
+    String comment = getComment(request);
+
+    Comment comm = new Comment(username, comment, false);
```

Comment:
Nit: Name this "comment"

(General rule of thumb - be lazy when naming variables, but not too lazy. Just lazy enough to make it readable. It's an easy assumption to just name a variable the name of its class, in these types of cases.)

Reply:
Changed